cvat-ui/src/components/analytics-page/analytics-page.tsx

```diff
@@ -23,12 +23,14 @@ import CVATLoadingSpinner from 'components/common/loading-spinner';
 import GoBackButton from 'components/common/go-back-button';
 import AnalyticsOverview, { DateIntervals } from './analytics-performance';
 import TaskQualityComponent from './task-quality/task-quality-component';
+import TaskConsensusComponent from './task-consensus/task-consensus-component';
 
 const core = getCore();
 
 enum AnalyticsTabs {
     OVERVIEW = 'overview',
     QUALITY = 'quality',
+    CONSENSUS = 'consensus',
```

Comment:
Why do we have consensus tab on the analytics page? As we discussed, it should be separate page

Reply:
Ah, I added this just for testing purpose. Anyway there's a seprate page for this.